src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
This class should not extend `Figure` if `Figure` is an abstract class that sets behavior, as per the checklist. Consider using an interface to define common behavior and ensure `Rectangle` implements this interface.